src/screens/NewAnalytics/Graphs/BarGraph/BarGraphTypes.res

```diff
@@ -0,0 +1,85 @@
+type \"type" = string
+type spacingLeft = int
+type spacingRight = int
+
+type categories = array<string>
+type align = string
+type color = string
+type gridLineWidth = int
+type gridLineColor = string
+type gridLineDashStyle = string
+type tickmarkPlacement = string
+type endOnTick = bool
+type startOnTick = bool
+type tickInterval = int
+type tickWidth = int
+type min = int
+type max = int
+type showInLegend = bool
+type name = string
+
```

Comment:
why separate types?
Any specific usecase? 
I'm not able to find anywhere these used and if the type can be directly placed

Reply:
we are having some types common to LineGraph and BarGraph which we will move it to common place